### DISCUSSION TITLE
Tokyo Night variant

### DISCUSSION THREAD

**Discussion by Scraft161** [promitheas17j/lemmy-ui-themes#4] [Ideas]
(migrated to GH by request, see the original [lemmy thread](https://iusearchlinux.fyi/comment/166472) for historical dicussion)

Add a tokyo night variant of the theme.

**Comment by Scraft161** [comment]
just asking if there are any updates regarding this.

**Reply by promitheas17j** [reply]
Still working on it, but due to work and uni ramping up and having to get a new PC its paused. Will finish it though.
